fcrepo-api-x-routing/README.md

```diff
@@ -0,0 +1,57 @@
+# API-X Routing module
+
+This module contains the API-X [routing](../src/site/markdown/execution-and-routing.md#routing) and [generic](../src/site/markdown/execution-and-routing.md#generic-endpoint-proxy) [implementations](../src/site/markdown/execution-and-routing.md#generic-intercepting-proxy) of execution components.
+
+## OSGi Deployment
+
+This module is automatically deployed when the `fcrepo-api-x` feature is installed in Karaf.
+
+## Configuration
```

Comment:
I think that some of these configuration parameters affect the implementation in nuanced ways, while other parameters are less subtle.  Regardless, it can be difficult to understand how these configuration parameters affect the implementation, and what needs to be changed (or not) for a typical installation.

I'm not trying to hold back the tagging of the code, but I think it would be helpful - and I think these parameters deserve - some exposition.  If not now, in the future.